Dockerfile

```diff
@@ -1,8 +1,21 @@
 FROM golang:1.13.3-stretch as builder
 
-# Username and password to use basic auth and download the repo.
-ARG NIGHTFALL_GITHUB_USER
-ARG NIGHTFALL_GITHUB_PASS
-# Verify username/pass args were passed in
-RUN test -n "$NIGHTFALL_GITHUB_USER"
-RUN test -n "$NIGHTFALL_GITHUB_PASS"
+ARG NIGHTFALL_DLP_RELEASE="v0.0.1"
+
+# create /projects and install DLP repo there
+WORKDIR /projects
+RUN git clone --branch $NIGHTFALL_DLP_RELEASE https://github.com/nightfallai/jenkins_test.git
+
+# navigate to the nightfalldlp repo and install it to $GOPATH/bin/nightfalldlp
+WORKDIR /projects/jenkins_test
```

Comment:
why are we hardcoding this?

Reply:
i'm not sure I understand the alternative. Do you just want this path behind a var?